plugins/imjournal/imjournal.c

```diff
@@ -761,6 +799,10 @@ CODESTARTsetModCnf
 			free(fac);
 		} else if (!strcmp(modpblk.descr[i].name, "usepidfromsystem")) {
 			cs.bUseJnlPID = (int) pvals[i].val.d.n;
+			errmsg.LogError(0, RS_RET_DEPRECATED,
```

Comment:
parser_errmsg() might be more appropriate here, as it automatically outputs the error location